cms/djangoapps/course_creators/tests/test_admin.py

```diff
@@ -104,7 +104,7 @@ def check_admin_message_state(state, expect_sent_to_admin, expect_sent_to_user):
             # message sent. Admin message will follow.
             base_num_emails = 1 if expect_sent_to_user else 0
             if expect_sent_to_admin:
-                context = {'user_name': "test_user", 'user_email': 'test_user+courses@edx.org'}
+                context = {'user_name': "test_user", 'user_email': u'test_user+courses@edx.org'}
```

Comment:
I included this change in https://github.com/edx/edx-platform/pull/7131/, but it failed, so I assume that this is related to the 1.5 upgrade, and should not be included.